rudra/deployments/emr_scripts/maven_emr.py

```diff
@@ -23,9 +23,11 @@ def run_job(self, input_dict):
         log_file_name = '{}.log'.format(name)
 
         log_uri = 's3://{bucket}/{log_file}'.format(
-            bucket='{}-automated-analytics-spark-jobs'.format(self.env),
+            bucket=self.bucket_name,
             log_file=log_file_name)
 
+        logger.info("Logs are gonna store at {}".format(log_uri))
```

Comment:
Minor nitpick, this is grammatically incorrect 🙃, please change it to "logs will be stored at".

Reply:
ahh Got it !